zuul-core/src/main/java/com/netflix/config/PatternListStringProperty.java

```diff
@@ -38,7 +38,7 @@ public PatternListStringProperty(String name, String defaultValue) {
     protected List<Pattern> derive(String value) {
         ArrayList<Pattern> ptns = new ArrayList<>();
         if (value != null) {
-            for (String ptnTxt : value.split(",")) {
+            for (String ptnTxt : value.split(",", -1)) {
```

Comment:
I had to look this one up - using -1 (instead of the default of 0). Preserves trailing empty strings. I guess it's more explicit?

Reply:
yeah, https://errorprone.info/bugpattern/StringSplitter has some examples